### PR TITLE
Add logger

### PR DESCRIPTION
**Pull request by lamchuntat**
@ThorpeJosh @siewyeng @changukshin 
Need some advice for this PR. The logger that I added to `__main__.py` cannot be used in other modules, ie. only log messages from `__main__.py` are recorded, log messages from other modules couldn't be seen in stdout nor the log file :/

### CODE REVIEW

**Comment by lamchuntat** [comment]
Perhaps this link could be useful: https://docs.python.org/3/howto/logging.html#logging-from-multiple-modules

File: tp_timesheet/submit_form.py [hunk 1]
```diff
@@ -1,9 +1,13 @@
 """ Module containing tool to submit the form """
+import logging
 import os
 import datetime
 from selenium import webdriver
 from PIL import Image
 
+logger = logging.getLogger(__name__)
+logger.info("HELLOOOOOOO")
```
Comment (ThorpeJosh) [inline tp_timesheet/submit_form.py:9]: Same here

**Comment by ThorpeJosh** [comment]
Lastly just from the docs, I believe they all should be referencing the same logger if you don't want seperate log files or behaviour for different modules. So the `getLogger()` call should have same name as the logger in `__main__.py` where you are attaching the handlers.

**Comment by lamchuntat** [comment]
Shifted setup for logger into the `__init__.py` file. Loggers in other modules are now working with the same configuration. I've also replaced all existing print statements with logs.

@ThorpeJosh @siewyeng @changukshin please advise me on whether there will be any potential issues with setting up the logger in this way and if the logging levels are appropriate for the print statements they are replacing. Any other suggestions are greatly welcomed.

File: tp_timesheet/__init__.py [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+log_format = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+
+# log to stdout
+stream_handler = logging.StreamHandler()
+stream_handler.setLevel(logging.DEBUG)
+stream_handler.setFormatter(log_format)
+logger.addHandler(stream_handler)
+
+# log to file
+log_dir = Path.home().joinpath(".config", "tp-timesheet.log")
```
Comment (ThorpeJosh) [inline tp_timesheet/__init__.py:21]: log should maybe go inside the `.config/tp-timesheet/logs/` directory so it doesn't fill up the users `.config`

**Review by ThorpeJosh — CHANGES_REQUESTED** [review]

File: tp_timesheet/__init__.py [hunk 3]
```diff
@@ -1,2 +1,25 @@
...
+file_handler = logging.FileHandler(log_dir)
```
Comment (ThorpeJosh) [inline tp_timesheet/__init__.py:22]: You ok to try the rotation logs instead of one log file?

Maybe rotate logs every month?

Comment (lamchuntat) [reply tp_timesheet/__init__.py:22]: Yup let's do that, every month sounds good

**Comment by lamchuntat** [comment]
@ThorpeJosh @siewyeng @changukshin 

Finally managed to get the checks to pass 😮‍💨 Could you help me review and merge if all's good?

**Review by ThorpeJosh — CHANGES_REQUESTED** [review]
Looks good.

Previously we controlled the "debug" print statements with:
```python
if args.verbose:
     print('log message')
```
But now we want to replace them all with simply `logger.debug('log message')` and instead set the `stream_handler` level to INFO or DEBUG according to the args.verbose. This way the debug messages will always go to the log file but not necessarily to stdout.

**Pull request merged**